aten/src/ATen/core/interned_strings.h

```diff
@@ -67,14 +67,19 @@ namespace c10 {
   _(prim, ConstantChunk)           \
   _(prim, NoneGenerator)           \
   _(aten, floordiv)                \
-  _(prim, MemoryFence)             \
   _(prim, fork)                    \
   _(prim, RaiseException)          \
   _(aten, append)                  \
   _(aten, format)                  \
   _(aten, __not__)                 \
   _(aten, __is__)                  \
   _(aten, __isnot__)               \
+  _(aten, _assign)                 \
+  _(aten, _set_item)               \
```

Comment:
For LHS indexing, couldn't we desugar it into `copy_` like pytorch does it, and essentially copy & paste https://github.com/pytorch/pytorch/blob/master/torch/csrc/autograd/python_variable_indexing.cpp#L332-L381 ? I wrote the RHS JIT code by copying the existing RHS code under the assumption that we could probably template-ize the JIT indexing code and the pytorch indexing code one day

Reply:
I think what I have now is functionally equivalent since the rvalue overload of Tensor assignment will before a copy. It will make things more explicit to directly invoke copy_ though.